### PR TITLE
Add separate naomi project page

### PR DESCRIPTION
**Pull request by r-ash**
This PR will

* Move `projects` article to be landing page of projects section
* Add a new page for `naomi`

I'm expecting to update the png of the poster when we have settled on the final version. I should probably compress this too? 1MB is quite big for an image on a web page?

### CODE REVIEW

File: content/projects/_index.md [hunk 1]
```diff
@@ -25,15 +25,11 @@ how close they are to the [UNAIDS 90-90-90 target](https://www.unaids.org/en/res
 
 * [Paper describing the model](https://www.biorxiv.org/content/10.1101/532010v1)
 
-### Naomi
+### [Naomi](/projects/naomi)
```
Comment (richfitz) [inline content/projects/_index.md]: This is too subtle in the page.  Something like `see the [project page][/projects/naomi]` for more information, though `[naomi]` would probably work for the link

File: content/projects/naomi.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
++++
+date = "2020-02-03"
+title = "Naomi"
++++
+
+[https://naomi.unaids.org](https://naomi.unaids.org/)
+
+A web interface for a model estimating various HIV epidemic indicators at a regional level. For use by
```
Comment (richfitz) [inline content/projects/naomi.md]: We should link to Jeff somewhere here if he has a web presence. I think the poster would be better displayed as a thumbnail linking through to a pdf?

File: content/projects/naomi.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
++++
+date = "2020-02-03"
+title = "Naomi"
++++
+
+[https://naomi.unaids.org](https://naomi.unaids.org/)
+
+A web interface for a model estimating various HIV epidemic indicators at a regional level. For use by 
+countries in association with UNAIDS. The app is comprised of:
```
Comment (richfitz) [inline content/projects/naomi.md]: Can we say "this was used in workshops with UNAIDS in December 2019-January 2020, with ongoing direct use by countries" to give a better indication of the use.  Do we want to get Jeff to provide us with a 2-sentence blurb about the science too?

**Review by richfitz — CHANGES_REQUESTED** [review]

File: content/projects/naomi.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+* [an R API and R workers](https://github.com/mrc-ide/hintr), managed by a [Redis queue](https://github.com/mrc-ide/rrq)
+* [a Kotlin web server](https://github.com/mrc-ide/hint)
+* [a Vue.js front-end](https://github.com/mrc-ide/hint/tree/master/src/app/static)
+* [HIV model](https://github.com/mrc-ide/naomi)
+
+Presented at a poster [RSLondonSouthEast](https://rslondon.ac.uk/) on 6th February
```
Comment (hillalex) [inline content/projects/naomi.md]: "at" -> "as"? re image, better to just link it as a pdf than embed as a png.

**Comment by r-ash** [comment]
Still waiting for some sentences from Jeff and to see if he wants us to link to his twitter or imperial bio or something

File: content/projects/naomi.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+A web interface for a model estimating various HIV epidemic indicators at a regional level. The application is used by teams of health officials from sub-Saharan African countries to inform HIV programme targets. The tool was rolled out during a series of UNAIDS-led workshops in Johannesburg and Abidjan from December 2019 - January 2020. The model produces district-level estimates of people living with HIV, coverage of and unmet need for antiretroviral treatment, and new HIV infections. Estimates are used to develop official HIV programme targets and budget allocations for the US Government PEPFAR programme planning, Global Fund grant applications, and other national HIV programme planning needs.
+
+The app is comprised of:
+
+* [HIV model](https://github.com/mrc-ide/naomi)
+* [wrapped by an R API](https://github.com/mrc-ide/hintr)
+* [a Kotlin web server](https://github.com/mrc-ide/hint)
+* [a Vue.js front-end](https://github.com/mrc-ide/hint/tree/master/src/app/static)
+* Supported by
+   * [web application database](https://github.com/mrc-ide/hint-db)
```
Comment (richfitz) [inline content/projects/naomi.md]: "web application database" -> "a Postgres database for the Kotlin web app"

File: content/projects/naomi.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+[https://naomi.unaids.org](https://naomi.unaids.org/)
+
+A web interface for a model estimating various HIV epidemic indicators at a regional level. The application is used by teams of health officials from sub-Saharan African countries to inform HIV programme targets. The tool was rolled out during a series of UNAIDS-led workshops in Johannesburg and Abidjan from December 2019 - January 2020. The model produces district-level estimates of people living with HIV, coverage of and unmet need for antiretroviral treatment, and new HIV infections. Estimates are used to develop official HIV programme targets and budget allocations for the US Government PEPFAR programme planning, Global Fund grant applications, and other national HIV programme planning needs.
+
+The app is comprised of:
+
+* [HIV model](https://github.com/mrc-ide/naomi)
+* [wrapped by an R API](https://github.com/mrc-ide/hintr)
+* [a Kotlin web server](https://github.com/mrc-ide/hint)
```
Comment (richfitz) [inline content/projects/naomi.md:14]: "web server" or "web app"?

**Review by richfitz — APPROVED** [review]
Suggestions, but otherwise LGTM.  I'd be happy to merge and deploy, then add Jeff's comments when he has time to send them

**Pull request merged**